packages/cli/src/zosfiles/upload/dtu/DirToUSSDir.handler.ts

```diff
@@ -54,12 +54,19 @@ export default class DirToUSSDirHandler extends ZosFilesBaseHandler {
             uploadOptions.filesMap = this.buildFilesMap(commandParameters);
         }
 
-        const uploadApi = commandParameters.arguments.recursive ? Upload.dirToUSSDirRecursive : Upload.dirToUSSDir;
-        const response = await uploadApi.bind(Upload)(session, inputDir, commandParameters.arguments.USSDir, uploadOptions);
+        commandParameters.response.progress.startBar("Uploading files...");
```

Comment:
The `startBar` method expects an `ITaskWithStatus` object rather than a string. Please update this to pass in the task object already defined in this handler:
```suggestion
        commandParameters.response.progress.startBar({ task: status });
```